transactions/src/test/java/org/jboss/hal/testsuite/test/transactions/TransactionCommitAndRollbackTestCase.java

```diff
@@ -102,18 +106,39 @@ public void before() {
             "AAAAAAAAAAAAAP__CigFZtXWO0pXxVwMAAAFsgAAAAEAAAAAAAAAAAAAAAAAAP__CigFZtXWO0pXxVwMAAAFrTEHAgIA"
     };
 
-    private static final Vector<PreparedTransactionTestCell> TRANSACTIONS_TEST_DATA = new Vector<PreparedTransactionTestCell>() {
+    private static final Vector<PreparedTransactionTestCell> GUI_TRANSACTIONS_TEST_DATA = new Vector<PreparedTransactionTestCell>() {
         {
         add(new PreparedTransactionTestCell(TRANSACTIONS_XID[0], PerformedAction.ROLLBACK));
         add(new PreparedTransactionTestCell(TRANSACTIONS_XID[1], PerformedAction.COMMIT));
         add(new PreparedTransactionTestCell(TRANSACTIONS_XID[2], PerformedAction.COMMIT));
         add(new PreparedTransactionTestCell(TRANSACTIONS_XID[3], PerformedAction.ROLLBACK));
         add(new PreparedTransactionTestCell(TRANSACTIONS_XID[4], PerformedAction.COMMIT));
-        add(new PreparedTransactionTestCell(TRANSACTIONS_XID[5], PerformedAction.ROLLBACK));
-        add(new PreparedTransactionTestCell(TRANSACTIONS_XID[6], PerformedAction.COMMIT));
         }
     };
 
+    private static final int[] REFRESH_TEST_INDEXES = {5, 6};
+
+    @Test
+    public void rollbackTransactionsInCLIAndRefresh() throws IOException {
+        String[] testedXIDs = {TRANSACTIONS_XID[REFRESH_TEST_INDEXES[0]], TRANSACTIONS_XID[REFRESH_TEST_INDEXES[1]]};
+        Assert.assertNotNull(page.getResourceManager().getResourceTable().getRowByText(0, testedXIDs[0]));
+        Assert.assertNotNull(page.getResourceManager().getResourceTable().getRowByText(0, testedXIDs[1]));
+
+        Assert.assertTrue("Commit in CLI did not went well! Is the base64 xID right?",
+                operations.invoke("commit-prepared-transaction",
+                DEFAULT_MESSAGING_SERVER_ADDRESS, Values.of("transaction-as-base-64", testedXIDs[0])).booleanValue());
+        Assert.assertTrue("Commit in CLI did not went well! Is the base64 xID right?",
+                operations.invoke("commit-prepared-transaction",
+                DEFAULT_MESSAGING_SERVER_ADDRESS, Values.of("transaction-as-base-64", testedXIDs[1])).booleanValue());
+
+        page.refresh();
+
+        Assert.assertNull("Committed transaction should not be present in table after refresh",
+                page.getResourceManager().getResourceTable().getRowByText(0, testedXIDs[0]));
+        Assert.assertNull("Committed transaction should not be present in table after refresh",
+                page.getResourceManager().getResourceTable().getRowByText(0, testedXIDs[1]));
+    }
```

Comment:
I suggest to add some assert that some prepared transactions remain in the table after the refresh.